base/scopedvalues.jl

```diff
@@ -40,17 +88,21 @@ julia> sval[]
     Scoped values were introduced in Julia 1.11. In Julia 1.8+ a compatible
     implementation is available from the package ScopedValues.jl.
 """
-mutable struct ScopedValue{T}
-    # NOTE this struct must be defined as mutable one since it's used as a key of
```

Comment:
Should these comment be preserved? They seem like important invariants.

Reply:
Well they were intentionally added by @aviatesk in https://github.com/JuliaLang/julia/pull/52372, so that's why I'm asking :)

Reply:
Added back (since you said you didn't mind either way @Keno)